src/image_resources/centos8-appstream.repo

```diff
@@ -1,5 +1,5 @@
 [centos8-appstream]
 name=CentOS-8-Appstream
 baseurl=http://mirror.centos.org/centos/8/AppStream/x86_64/os/
-enabled=0
```

Comment:
Why did you enable centos repos by default?. We left them disabled to prevent RPM conflicts.
You can enable them with dnf passing either --enablerepo=centos8 or --enablerepo=centos8-appstream